calcom/cal.com
# fix: /getting-started/connected-calendar Toggle the first calendar on

## What does this PR do?

[CAL-115] /getting-started/connected-calendar Toggle the first calendar on

Fixes #4384 

https://github.com/calcom/cal.com/assets/37188392/aa9a0179-a355-464d-84c6-15fde1b07762


**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

1. Create a new account
2. Follow the steps and connect any calendar
3. You should see the first calendar selected by default

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
